runbotics/runbotics-scheduler/src/websocket/bot/process-launch/bot-process-instance-event.service.ts

```diff
@@ -26,6 +26,10 @@ const COMPLETED_UPDATE_FIELDS = [
 ];
 const STARTED_UPDATE_FIELDS = ['input', 'created'];
 
+function setWarning(processInstanceEvent: IProcessInstanceEvent): boolean {
+    return processInstanceEvent.step === 'event.warning';
```

Comment:
IMO we should have a single source of truth for elements like `event.warning`

Reply:
the name `setWarning` sounds like a setter in a class or manipulation of the parameter. A better term would be `hasWarning` (in the ideal world I would include the name `processInstance` in it but we can deduct if from parameter)